Assets/Scripts/AiPatrol.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AiPatrol : MonoBehaviour
+{
+    public float walkSpead;
+    [HideInInspector]
+    public bool mustPetrol;
```

Comment:
Co znaczy, że ma mieć bęzyne?

Reply:
Rozumiem także że jak nie będzie paliwa to nie będzie chodził xd

Reply:
aj typo, poprawiam xd